graphics/graphviz/Portfile

```diff
@@ -284,29 +286,21 @@ post-destroot {
     # Make the configuration file that makes the plugins work.
     system "GVBINDIR=${destroot}${prefix}/lib/graphviz DYLD_LIBRARY_PATH=${destroot}${prefix}/lib ${destroot}${prefix}/bin/dot -c"
 }
-
-livecheck.type                  regex
```

Comment:
This seems to work:
```
livecheck.type                  regex
livecheck.url                   https://graphviz.org/download/source/
# For some reason some versions are listed as x.gz instead of x.tar.gz
livecheck.regex                 ${my_name}-(\[0-9\]+\\.\[0-9\]+(?:\\.\[0-9\]+)*)\\.(?:tar|gz)
```

Or using the GitLab releases API:
```
livecheck.type                  regex
livecheck.url                   https://gitlab.com/api/v4/projects/4207231/releases
livecheck.regex                 \"name\":\s*\"(\[0-9\]+\\.\[0-9\]+(?:\\.\[0-9\]+)*)\"
```